src/AWS.Deploy.Orchestration/DeploymentCommands/CdkDeploymentCommand.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AWS.Deploy.Common;
+using AWS.Deploy.Common.Recipes;
+using AWS.Deploy.Orchestration.DisplayedResources;
+
+namespace AWS.Deploy.Orchestration.DeploymentCommands
+{
+    public class CdkDeploymentCommand : IDeploymentCommand
+    {
+        public DeploymentStatus Status => throw new NotImplementedException();
```

Comment:
I assume this will eventually be implemented.

Reply:
Synced offline and concluded that this field is not required. Hence removed it.